package.json

```diff
@@ -25,19 +25,18 @@
   },
   "scripts": {
     "clean": "rimraf ./dist",
-    "content": "node src/scripts/build-content-tree.js ./src/content ./src/_content.json",
     "prestart": "npm run clean",
     "start": "NODE_ENV=development webpack-dev-server --config webpack.dev.js --env.dev",
-    "prebuild": "npm run clean",
-    "build": "npm run content && npm run fetch && NODE_ENV=production webpack --config webpack.prod.js",
+    "content": "node src/scripts/build-content-tree.js ./src/content ./src/_content.json",
     "fetch": "sh src/scripts/fetch.sh",
+    "prebuild": "npm run clean",
+    "build": "npm run content && npm run fetch && NODE_ENV=production webpack --config webpack.prod.js && npm run sitemap && echo webpack.js.org > dist/CNAME",
```

Comment:
Shouldn't _fetch_ run before content so the external markdown files are treated as content routes?

Reply:
Not sure, let me test

Reply:
Not, it is not the case. Routes are not created.

Reply:
But, we don't need those routes? Or they aren't being created and that's a bug?

Reply:
No, we don't need them. I have compared each page to webpack.js.org and everything is there.

Reply:
I think the problem is, that `clean` doesnt remove fetched content, probably that is why you still get contents

Reply:
which should be the case for a clean build

Reply:
@EugeneHlushko I am not following you :D

Reply:
If you run `fetch` after `content` after pulling the repo, you wont get pages from fetch into directory tree

Reply:
content should be run after fetch

Reply:
@jeremenichelli @EugeneHlushko Absolutely right. Fixed in https://github.com/webpack/webpack.js.org/pull/2254/commits/fef8397c65a22dde49bfffbc6a97ccb73538aa4f